types/fsp.go

```diff
@@ -22,6 +22,8 @@ import (
 )
 
 const (
+	// WaitFillFsp is fsp need fill by caller if they want to use fsp.
+	WaitFillFsp = -2
```

Comment:
When will the caller want to us fsp?

Reply:
- -~ the old way needs GetDuration caller to check whether `result.fsp == WaitFillFsp` than refill Fsp or do nothing...that's buggy.

Now force pass `fillFsp` paramter in `GetDuration` and no longer need this flag. PTAL~thx